Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM node:6-onbuild
 MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
-RUN npm run install:clean --production
```

Comment:
Looking at https://github.com/nodejs/docker-node/blob/10940306b6ae14f9d2fb0d9a7327e768eadc039a/6.1/onbuild/Dockerfile it looks like most of these steps already happen so this file could probably be shortened to:

```
FROM node:6-onbuild
MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
EXPOSE 5465
```

(moving the environment variables out of the image)